doc/community-build/docker/cortx-all/README.md

```diff
@@ -1,4 +1,4 @@
-# Generate CORTX-ALL container image
+# Generate CORTX container image
```

Comment:
Please `s/image/images` as we need two images now.

Reply:
Yes just now replaced this as well.